.ddev/config.yaml

```diff
@@ -0,0 +1,267 @@
+type: drupal
+docroot: web
+php_version: "8.3"
+webserver_type: nginx-fpm
+router_http_port: "80"
+router_https_port: "443"
+xdebug_enabled: false
+additional_hostnames:
+ - drupal-core-testing
```

Comment:
Adding this here means you'll have trouble running more than one project at a time.

Reply:
yes, but it makes easier to access phpunit tests output htmls.